src/utils/isEmpty.js

```diff
@@ -0,0 +1,8 @@
+const isEmpty = (value) => {
+    value === undefined
+      || value === null
+      || (typeof value === 'object' && Object.keys(value).length === 0)
+      || (typeof value === 'string' && value.trim().length === 0);
+  };
+  export default isEmpty;
+  
```

Comment:
I think the end of this line has a space bar, that's why it's still requesting for a new line.